final_project/dags/sql/customers_transfer_from_data_lake_raw_to_bronze.sql

```diff
@@ -0,0 +1,24 @@
+MERGE `de2022-stanislav-rezen.bronze.customers` main
```

Comment:
In most cases for data like customers I use a partitioned table in Raw/Bronze and a NOT partitioned table in Silver.
 I do not use MERGE in Raw/Bronze layer. I keep the full dump of each day in a partition. It helps to investigate issues, and anomalies, and do debugging. Of course, I use MERGE in Silver.